inc/plugin.class.php

```diff
@@ -1793,13 +1812,19 @@ static function getSpecificValueToDisplay($field, $values, array $options = [])
             $value = $values[$field];
             $ID = $value;
             $plugin = new self;
-            $plugin->checkStates();
             $plugin->getFromDB($value);
             $plug = $plugin->fields;
 
-            if (function_exists("plugin_".$plug['directory']."_check_config")) {
-               // init must not be called for incompatible plugins
-               self::load($plug['directory'], true);
+            $plugin->checkPluginState($plug['directory']);
+
+            $plugin_setup_file = GLPI_ROOT . '/plugins/' . $plug['directory'] . '/setup.php';
```

Comment:
This change has been made to prevent call of the init function of the plugin, which can lead to unexpected result if no check on plugin activation is done on this init function.